### ISSUE TITLE
cinnamon panel padding

### ISSUE THREAD

**Issue by jeweloper** [Antergos/Numix-Frost#32]
The cinnamon.css defines for both panels a padding of 2px.
Maybe there is a reason to set this padding for the bottom panel, but it makes a bad appearance for the top panel.

![unbenannt](https://cloud.githubusercontent.com/assets/18451155/14767671/e2409ba6-0a2b-11e6-890e-d31d10302708.jpg)

You can see, that the text is not in the centre of the panel.

enviroment:
numix-frost-themes  3.5.r679-1
cinnamon 2.8.8-2
Kernel 4.5.1-1-Arch

**Comment by thebeardedchild** [comment]
Also, and I'm not sure if this is an intentional change in Numix design, but the dark gray color of the panel in Cinnamon no longer seems to be #2d2d2d, and is much lighter than other instances across the theme.

**Comment by jeweloper** [comment]
@lots0logs wanted to help and create a pull request.
But then I realized, that the current numix-cinnamon version is published in /usr/share/themes/Numix-Frost/includes/cinnamon/Numix-Cinnamon/cinnamon. It seems the one published under /usr/share/themes/Numix-Frost/cinnamon is a old version. I refer to the package released under http://mirrors.antergos.com/antergos/x86_64/numix-frost-themes-3.5.r679-1-any.pkg.tar.xz

Is this a bug in your build process?

Btw: The reported issue does not occur in the current version.

**Comment by fyvfy** [comment]
After update to cinnamon 3.0.1:
And now "the text is not in the centre of the" bottom panel too (and icons). Menu, applications shortcuts and buttons(?) of running apps. Applets and system tray icons looks ok.

**Comment by ekmecic** [comment]
Reverting to version `3.5.r667-1` fixes both the padding problem and the color change problem that @thebeardedchild mentioned.

**Comment by jeweloper** [comment]
@lots0logs are you maybe missing the `--recursive` option?
Something like `git submodule update --init --recursive`?

**Comment by lots0logs** [comment]
No, I changed it back how it was actually. I remembered the reason why I had it that way in the first place :sweat_smile:  Anyway it should be okay now.

**Comment by lukaszjurczak** [comment]
![screenshot from 2016-05-10 19-29-59](https://cloud.githubusercontent.com/assets/8094867/15156092/b41db86e-16e5-11e6-898d-5a6681234409.png)

Fresh install and there is still problem with padding.

**Comment by ekmecic** [comment]
The issue still persists in the latest version.

**Comment by jeweloper** [comment]
The bug still exists in v3.6.4. 
@lots0logs Did Antergos/numix-cinnamon#3 fix the issue?
